projects/bitcoin-core/project.yaml

```diff
@@ -0,0 +1,19 @@
+homepage: "https://github.com/bitcoin/bitcoin"
+main_repo: 'https://github.com/bitcoin/bitcoin.git'
+language: c++
+primary_contact: "marco@chaincode.com"
+auto_ccs:
+  - "fanquake@gmail.com"
+  - "john@brink.dev"
+  - "jonas@chaincode.com"
+  - "laanwj@gmail.com"
+  - "pieter@chaincode.com"
```

Comment:
@sipa @laanwj (Adding you to CC on this pull requests, in case you haven't done so already)

Reply:
LGTM, thanks